modules/graceful-shutdown.adoc

```diff
@@ -7,6 +7,11 @@
 
 You can shut down your cluster in a graceful manner so that it can be restarted at a later date.
 
+[NOTE]
+====
+From the installation date, you can leave a cluster shut down for up to 1 year and expect it to restart gracefully. After which time, the cluster certificates expire.
```

Comment:
The way this is worded now doesn't seem super clear that it is 1 year from the installation date. Suggest a reword. Maybe?
```
You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire.
```
or maybe:
```
As long as you restart the cluster before the cluster certificates expire, which is a year from the installation date, you can expect the cluster to restart gracefully.
```